src/IconButton/IconButton.stories.tsx

```diff
@@ -70,6 +70,10 @@ export default {
       description: 'The disabled of the IconButton.',
       if: { arg: 'interactive' },
     },
+    darkMode: {
+      description: 'The colors of the IconButton are updated to the inverse color variant when dark mode is enabled.',
+      if: { arg: 'interactive' },
+    },
```

Comment:
the property name "darkMode" is misleading. This is solely inversing the color schema. It should not contain any reference to dark or light mode. preferred name would be "inverseColors"

Reply:
@felix-hcl, The property name has been updated to 'inversecolors' as suggested